result/isislsp.yaml

```diff
@@ -432,27 +461,326 @@ components:
             external:
               x-field-uid: 2
         prefix_attributes:
-          $ref: '#/components/schemas/IsisLsp.PrefixAttributes'
+          description: |-
+            Extended Prefix Attribute flags container sub-TLV is type 4.
```

Comment:
Need to discuss. Looks like backward compatibility breaking change.

Reply:
although these are not populated from the state machine till date. But reverting to previous name.

Reply:
done